domain-model/src/main/java/uk/gov/hmcts/cmc/domain/models/directionsquestionnaire/RequireSupport.java

```diff
@@ -0,0 +1,61 @@
+package uk.gov.hmcts.cmc.domain.models.directionsquestionnaire;
+
+import lombok.Builder;
+import lombok.EqualsAndHashCode;
+import org.apache.commons.lang3.builder.ReflectionToStringBuilder;
+import uk.gov.hmcts.cmc.domain.models.response.YesNoOption;
+
+import java.util.Optional;
+
+import static uk.gov.hmcts.cmc.domain.utils.ToStringStyle.ourStyle;
+
+@EqualsAndHashCode
+public class RequireSupport {
+
```

Comment:
Please change this to languageInterpreted or interpretedLanguage. Interpreter would imply person who does it. Also in CCD its correct.
I picked it up when I cherry picked this for my PR.

Thanks.